views/js/qtiCreator/widgets/helpers/validators.js

```diff
@@ -24,18 +24,18 @@ define([
 ], function(validators, _, __, Element) {
     'use strict';
 
-    var _qtiIdPattern = /^[_a-zA-Z]{1}[a-zA-Z0-9\-._]{0,31}$/i;
+    // var _qtiIdPattern = /^[_a-zA-Z]{1}[a-zA-Z0-9\-._]{0,31}$/i;
```

Comment:
@handleman , @AnastasiaArcadia and @ampaveliev , 
after talking to @div222 we have decided to comment out the current functionality of the validator as it behavior is misleading by sending the previous valid value ,
as a outcome of this,  next sprint we will have a bug ticket that will tackle this properly

Reply:
Ok. Maybe useful to add a comment because commented code always looks slightly confusing

Reply:
i actually wanted your input about this , because while minimizing i had the impression that it might break some functionality , so 
should i remove the code? 
should i leave it commented out but just add some details ?
should i use a different type of comment e.g /* here the data */

Reply:
>should i remove the code?
should i leave it commented out but just add some details ?

In my opinion, we should remove the code with a todo or comment it with a description that explains why the code wasn't removed.

> should i use a different type of comment e.g /* here the data */

No difference in this case. 

So, a short description should be enough for me :slightly_smiling_face: